app/views/entries/index.pdf.slim

```diff
@@ -4,6 +4,7 @@ ul
   li= "Project: #{@entry_finder.project_name}" if @entry_finder.project_name
   li= "Period: #{I18n.l(@entry_finder.from, format: :short)} - #{I18n.l(@entry_finder.to, format: :short)}"
   li= "Total: #{formatted_minutes(@entry_finder.total, legend: true)}"
+  li= "Collaborators: #{@entry_finder.collaborators.map(&:username).join(', ')}"
```

Comment:
or entry_finder_presenter - that would also be good place for @entry_finder.show_colllaborators?